### DISCUSSION TITLE
CoreFreq shows off

### DISCUSSION THREAD

**Discussion by cyring** [dankamongmen/notcurses#2684] [Show and tell]
This was inspired from old dos CRTC programming

[CoreFreq](https://github.com/cyring/CoreFreq) UI
* is made **without** Curses
* is designed for low latency
* prints what you see

![2022-10-22-121335_644x1012_scrot](https://user-images.githubusercontent.com/11563789/197333962-51f9acb2-df93-4690-8aae-c0fe57f65301.png)

* most compatible 7-bits ASCII with UTF-8
* in memory drawing with periodic fused
* old school double buffering

![2022-10-22-121443_644x1012_scrot](https://user-images.githubusercontent.com/11563789/197333964-cebe6272-ae3f-4b38-a10b-7dfafd6a717e.png)

**Comment by dankamongmen** [comment]
looks really nice!
